src/components/Checkout/TeamAttribution.vue

```diff
@@ -74,41 +74,40 @@ export default {
 	},
 	mounted() {
 		this.selectedId = this.teamId || 0;
+		this.updateLoanReservation();
 	},
 	methods: {
 		updateLoanReservation() {
-			if (this.selectedId !== this.teamId) {
```

Comment:
If we wait for the team ID to be defined in mounted, do we need to remove this check?

Reply:
Or is this just in case?

Reply:
Sounds like just in case, which I think is fine.